python/src/LinearModelTest_doc.i.in

```diff
@@ -354,3 +354,125 @@ Examples
 >>> print(test_result)
 class=TestResult name=Unnamed type=DurbinWatson binaryQualityMeasure=true p-value threshold=0.05 p-value=0.653603 description=[H0: auto.cor=0]
 "
+
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LinearModelTest::PartialRegression
+"Test whether two discrete samples are independent.
+
+**Available usages**:
+
+    LinearModelTest.PartialRegression(*firstSample, secondSample, selection*)
+
+    LinearModelTest.PartialRegression(*firstSample, secondSample, selection, level*)
+
+
+Parameters
+----------
+firstSample : 2-d sequence of float
+    First tested sample, of dimension :math:`n \geq 1`.
+secondSample : 2-d sequence of float
+    Second tested sample, of dimension 1.
+selection : sequence of int, maximum integer value :math:`< n`
+    List of indices selecting which subsets of the first sample will successively
+    be tested with the second sample through the regression test.
+level : positive float :math:`< 1`
+    Threshold p-value of the test (= first kind risk), it must be
+    :math:`< 1`, equal to 0.05 by default.
+
+Returns
+-------
+testResults : Collection of :class:`~openturns.TestResult`
+    Results for each component of the linear model including intercept.
+
+See Also
+--------
+LinearModelTest_FullRegression, LinearModelTest_LinearModelFisher
+
+Notes
+-----
+The Partial Regression Test is used to check the quality of the linear regression
+AnalyticalFmodel between two samples: *firstSample* of dimension *n* and *secondSample* of
+dimension 1. The parameter *selection* enables to select specific subsets of the
```

Comment:
AnalyticalFmodel?

Reply:
Btw you check a linearity between selected indices of first sample and second sample

Reply:
ok